source/api_documentation/transactions_api/_affiliate_optional_param_table.rst

```diff
@@ -0,0 +1,41 @@
+Additional Feature Parameters
+-----------------------------
+
+..  list-table::
+  :widths: 30 8 40
+  :header-rows: 1
+  :class: parameters
+
+  * - Field
+    - Name in Reports
+    - Description
+
+  * - bundled_destination_ids
+    - Bundled Destination Ids
+    - 
```

Comment:
Where is this description? Let's add one (likely there was in the dev portal, and maybe help text was missing it?)